about_sigils.exs

```diff
@@ -14,21 +14,20 @@ defmodule AboutSigils do
   end
 
   think "The ~S sigil is similar to ~s but doesn't do interpolation" do
-    assert_?  ~S[1 + 1 = #{1+1}] == ~s[1 + 1 = #{1+1}]
-    assert_?  ~S[1 + 1 = #{1+1}] == "1 + 1 = \#\{1+1\}"
+    assert ~s[1 + 1 = #{1+1}] == __?
+    assert ~S[1 + 1 = #{1+1}] == __?
```

Comment:
That's fair. I seem to have gotten confused reading this example, because now I don't really agree with the change... Maybe we should explicitly introduce escaping and then lean on that?